wandb/integration/sb3/__init__.py

```diff
@@ -0,0 +1,106 @@
+"""W&B callback for sb3
+
+Really simple callback to get logging for each tree
+
+Example usage:
+
+```python
+import gym
+import time
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import DummyVecEnv
+from wandb.integration.sb3 import WandbCallback
+import wandb
+
+config = {
+    "policy_type": "MlpPolicy",
+    "total_timesteps": 25000
+}
+experiment_name = f"PPO_{int(time.time())}"
+
+def make_env():
+    env = gym.make("CartPole-v1")
+    env = gym.wrappers.Monitor(env, f"videos/{experiment_name}", force=True) # record videos
+    env = gym.wrappers.RecordEpisodeStatistics(env) # record stats such as returns
+    return env
+
```

Comment:
Add a comment to highlight that wandb.init(..) must be called before you can use the WandbCallback. I'd prefer to over-educate than have people confused about why we're doing things :)

Reply:
done